WTWSMS/common/event_modifiers/WTWSMS_SPARTA.txt

```diff
@@ -0,0 +1,27 @@
+this_is_not_sparta = { # Spartan cultured character is a coward, AKA unthinkable in Spartan culture
+	icon = 90
+
+	general_opinion = -30
+	monthly_character_prestige = -1
+
+	martial = -20 # Prevents a Spartan AI ruler from choosing characters with the Craven Trait as Commanders
```

Comment:
They won't be picked if anyone is better than them. How can we prevent people with this Modifier from being picked at all?